manifests/generated/rbac.authorization.k8s.yaml.in

```diff
@@ -189,7 +189,7 @@ rules:
   - get
   - list
   - watch
-  - delete
+  - patch
```

Comment:
there is a reason why you remove the delete command from the RBAC?

Reply:
Yes, this was a deliberate change. I removed the permission to delete the VMI because I removed the need for it. Before this patch, virt-api would delete the running VMI to trigger a restart. With the introduction of (manual) RunStrategy, this no longer works because virt-controller wouldn't know how to respond in all cases. Instead the VM is patched with `RestartRequest` and virt-controller takes care of the rest. This is a much better approach overall anyways. Only one controller is changing the state of the VMI. Also, patch is idempotent -- so the virt-api controller doesn't need to constantly retry due to old versions of cached objects.